apps/test/unit/lib/levelbuilder/rubrics/LearningGoalItemTest.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {mount, shallow} from 'enzyme';
+import {expect} from '../../../../util/reconfiguredChai';
+import LearningGoalItem from '@cdo/apps/lib/levelbuilder/rubrics/LearningGoalItem';
+import sinon from 'sinon';
+
+describe('LearningGoalItem', () => {
+  let defaultProps;
+  let deleteItemSpy = sinon.spy();
+
+  beforeEach(() => {
+    defaultProps = {
+      deleteItem: deleteItemSpy,
+    };
+  });
+
+  it('renders correctly', () => {
+    const wrapper = mount(<LearningGoalItem {...defaultProps} />);
+    expect(wrapper.find('.uitest-learning-goal-card').length).to.equal(1);
+    expect(wrapper.find('.uitest-rubric-key-concept-input').length).to.equal(1);
+    expect(wrapper.find('input[type="checkbox"]').length).to.equal(1);
+    expect(wrapper.find('Button').length).to.equal(1);
+    expect(wrapper.find('textarea').length).to.equal(8);
+  });
+
+  it('toggles AI checkbox and enables editing of AI textboxes when checked', () => {
+    const wrapper = mount(<LearningGoalItem {...defaultProps} />);
+    const checkbox = wrapper.find('input[type="checkbox"]');
+    expect(wrapper.find('.ui-test-ai-prompt-textbox').length).to.equal(4);
+    expect(checkbox.prop('checked')).to.be.false;
+    expect(wrapper.find('.ui-test-ai-prompt-textbox').at(1).prop('disabled')).to
```

Comment:
Instead of checking these here, you can use `shallow` and add an assert like `expect(wrapper.find('EvidenceLevels').at(0).props().aiEnabled).to.be.true` then, in a test for `EvidenceLevels` check that these textboxes are enabled when `aiEnabled` is true (and vice versa)